lib/Doctrine/ORM/Cache/Logging/CacheLogger.php

```diff
@@ -19,69 +19,69 @@ interface CacheLogger
      * @param string         $regionName The name of the cache region.
      * @param EntityCacheKey $key        The cache key of the entity.
```

Comment:
This phpdoc is now quite tautological isn't it? Let's reconsider the comments?

Reply:
I've removed the descriptions.